### PR TITLE
feat(*): update labels for indian account creation and KYC india

### PR DESCRIPTION
**Pull request by blemoineovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12957
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
add a few text to precise that India users should check their information on subscription

## Related

:flags: Translation

- [ ] CMT-8414

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
**nitpick:** You are not authorised to modify `Messages_en_GB.json`

**Review by anooparveti — CHANGES_REQUESTED** [review]
This change must only be scoped to India users.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html [hunk 1]
```diff
@@ -33,6 +33,10 @@
         class="oui-tile__body"
         data-translate="user_account_identity_documents_intro"
     ></p>
+    <p
+        data-ng-if="$ctrl.isindian"
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html:37]:
```suggestion
        data-ng-if="$ctrl.isIndianSubsidiary"
```

File: packages/manager/apps/sign-up/src/form/translations/Messages_en_GB.json
Comment (anooparveti) [inline packages/manager/apps/sign-up/src/form/translations/Messages_en_GB.json:1]: Changes on this should come from translation process. You can exclude this file from the PR.

File: packages/manager/apps/sign-up/src/form/translations/Messages_fr_CA.json
Comment (anooparveti) [inline packages/manager/apps/sign-up/src/form/translations/Messages_fr_CA.json:1]: Changes on this should come from translation process. You can exclude this file from the PR.

**Review by anooparveti — CHANGES_REQUESTED** [review]
From my understanding, we need to replace an existing message to the Indian users. In the PR, I see that the new message is added, you also need to hide the existing one for Indian users.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html [hunk 2]
```diff
@@ -31,8 +31,14 @@
     <!--My Identity document description-->
     <p
         class="oui-tile__body"
+        data-ng-if="!$ctrl.isIndianSubsidiary"
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html:34]: From the ticket description, I don't think we should hide this message for Indian customers

File: packages/manager/apps/sign-up/src/form/translations/Messages_fr_CA.json
Comment (blemoineovh) [reply packages/manager/apps/sign-up/src/form/translations/Messages_fr_CA.json:1]: removed

File: packages/manager/apps/sign-up/src/form/translations/Messages_en_GB.json
Comment (blemoineovh) [reply packages/manager/apps/sign-up/src/form/translations/Messages_en_GB.json:1]: removed

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html (hunk 2 above)
Comment (anooparveti) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html:34]: @JacquesLarique the ticket says that we need to replace the current message with a new one for Indian Customers. So, I dont see any issue with the current implementation.

**Review by anooparveti — CHANGES_REQUESTED** [review]
Ticket asks for 2 changes,
1. Add a new message on the `Identity documents` page
2. Modify a message on Step 2 on the `Sign-up` page for the Indian Sub customers.

Current changes are done in an opposite manner. Changes on Identity Documents are scoped to Indian Sub users but the changes on Sign-up page are scoped to all users. It should be opposite.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html (hunk 2 above)
Comment (blemoineovh) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html]: ok i removed the condition

**Comment by blemoineovh** [comment]
> Ticket asks for 2 changes,
> 
>     1. Add a new message on the `Identity documents` page
> 
>     2. Modify a message on Step 2 on the `Sign-up` page for the Indian Sub customers.
> 
> 
> Current changes are done in an opposite manner. Changes on Identity Documents are scoped to Indian Sub users but the changes on Sign-up page are scoped to all users. It should be opposite.

hi @anooparveti i asked Abessiah and she said that both sentences changes are only for indian customer and that all others should have the same sentences like before,

" si on résume : on ne remplace ces 2 phrases que pr les clients INDIEN"

 i made a push to reflect it :)

File: packages/manager/apps/sign-up/src/form/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -6,5 +6,6 @@
   "sign_up_form_next_button": "Étape suivante",
   "sign_up_form_submit_button": "Créer un compte",
   "sign_up_form_submit_error": "Oups ! Une erreur est survenue lors de la création du compte.",
-  "sign_up_form_global_information": "Pour pouvoir commander, n'oubliez pas de renseigner toutes vos informations."
+  "sign_up_form_global_information": "Pour pouvoir commander, n'oubliez pas de renseigner toutes vos informations.",
+  "sign_up_form_global_information_india": "N'oubliez pas de saisir toutes les informations, et assurez-vous que vos données personnelles correspondent au nom et à l'adresse figurant dans vos documents officiels."
```
Comment (JacquesLarique) [inline packages/manager/apps/sign-up/src/form/translations/Messages_fr_FR.json:10]: The ticket state the sentence should be:
"N'oubliez pas de saisir toutes les informations, et assurez-vous que vos données personnelles correspondent au nom et à l'adresse figurant dans vos documents officiels, afin de pouvoir passer une commande."

File: packages/manager/apps/sign-up/src/form/form.controller.js [hunk 4]
```diff
@@ -111,6 +111,8 @@ export default class SignUpFormAppCtrl {
     this.loading.init = true;
 
     const { ovhSubsidiary } = this.$location.search();
+    this.isIndianSubsidiary = ovhSubsidiary;
```
Comment (JacquesLarique) [inline packages/manager/apps/sign-up/src/form/form.controller.js:114]: I have some doubt `ovhSubsidiary` (`this.$location.search()`) is equivalent to `isIndianSubsidiary`

**Review by JacquesLarique — CHANGES_REQUESTED** [review]

**Review by qpavy — APPROVED** [review]

File: packages/manager/apps/sign-up/src/form/form.controller.js (hunk 4 above)
Comment (blemoineovh) [reply packages/manager/apps/sign-up/src/form/form.controller.js]: yup, fixed using coreconfig service

File: packages/manager/apps/sign-up/src/form/translations/Messages_fr_FR.json (hunk 3 above)
Comment (blemoineovh) [reply packages/manager/apps/sign-up/src/form/translations/Messages_fr_FR.json]: updated

File: packages/manager/apps/sign-up/src/form/form.controller.js [hunk 5]
```diff
@@ -110,9 +111,12 @@ export default class SignUpFormAppCtrl {
     this.saveError = null;
     this.loading.init = true;
 
-    const { ovhSubsidiary } = this.$location.search();
-    if (ovhSubsidiary && ovhSubsidiary.match(/^[\w]{2}$/)) {
-      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, ovhSubsidiary);
+    this.$location.search();
```
Comment (JacquesLarique) [inline packages/manager/apps/sign-up/src/form/form.controller.js:114]: I don't understand the need for this

File: packages/manager/apps/sign-up/src/form/form.controller.js [hunk 6]
```diff
@@ -110,9 +111,12 @@ export default class SignUpFormAppCtrl {
...
+    this.isIndianSubsidiary = this.user.ovhSubsidiary;
```
Comment (JacquesLarique) [inline packages/manager/apps/sign-up/src/form/form.controller.js:115]: This is still not what is need, you need to check if ovhSubsidiary === 'IN' (of course do not use 'IN' but rather a constant)

Comment (blemoineovh) [reply packages/manager/apps/sign-up/src/form/form.controller.js]: updated to search for a constant

File: packages/manager/apps/sign-up/src/form/form.controller.js (hunk 5 above)
Comment (JacquesLarique) [reply packages/manager/apps/sign-up/src/form/form.controller.js:115]: As shared with you yesterday, either you keep the retrieval of subsidiary using `this.$location.search()` or you use `coreConfig`.
Since we use `this.$location.search()` to setItem in the localStorage, I would suggest to keep the code as it was initially and add `this.isIndianSubsidiary = ovhSubsidiary === SUBSIDIARIES.IN;`

**Review by JacquesLarique — CHANGES_REQUESTED** [review]

File: packages/manager/apps/sign-up/src/form/form.controller.js [hunk 7]
```diff
@@ -110,9 +112,12 @@ export default class SignUpFormAppCtrl {
...
 
-    const { ovhSubsidiary } = this.$location.search();
-    if (ovhSubsidiary && ovhSubsidiary.match(/^[\w]{2}$/)) {
-      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, ovhSubsidiary);
+    this.$location.search();
+    this.isIndianSubsidiary = this.user.ovhSubsidiary === SUBSIDIARIES.IN;
+
+    const locationSearch = this.$location.search();
+    if (locationSearch && locationSearch.match(/^[\w]{2}$/)) {
+      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, locationSearch);
```
Comment (qpavy) [inline packages/manager/apps/sign-up/src/form/form.controller.js:120]: Why did you change this part ?

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "user_account_identity_documents": "Mes documents d'identité",
   "user_account_identity_documents_sub_title": "Déposez vos documents",
-  "user_account_identity_documents_intro": "Vous avez créé votre compte et vous souhaitez faire une commande depuis notre site Indien. La règlementation en vigueur nécessite une vérification de votre identité avant toute commande.",
+  "user_account_identity_documents_intro": "Vous avez créé votre compte et vous souhaitez faire une commande depuis notre site Indien. La règlementation en vigueur nécessite une vérification de votre identité avant toute commande.<br />Assurez-vous que vos données personnelles correspondent au nom et à l'adresse figurant dans vos documents officiels.",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:4]: **nitpick:** Use a different key value pair in translation and add a break-line character in HTML.

File: packages/manager/apps/sign-up/src/constants.js [hunk 9]
```diff
@@ -23,6 +23,11 @@ export const HOME_PAGE = {
   WS: 'https://www.ovh.com/us/es/',
 };
 
+export const SUBSIDIARIES_LABEL_SUFFIX = {
+  IN: '_india',
+  DEFAULT: '',
+};
```
Comment (anooparveti) [inline packages/manager/apps/sign-up/src/constants.js:29]: **nitpick:** This implementation is not scalable. Lets say if we want to display the same message for some other sub, we will be in a position to copy-paste the existing trans for other sub.

**Review by anooparveti — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

**Review by acavalloni — APPROVED** [review]
